internal/xds/translator/listener.go

```diff
@@ -86,6 +87,7 @@ func (t *Translator) addXdsHTTPFilterChain(xdsListener *listener.Listener, irLis
 				RouteConfigName: irListener.Name,
 			},
 		},
+		UseRemoteAddress: &wrappers.BoolValue{Value: true},
```

Comment:
can we add this link https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for in the code for reference ?

Reply:
done